typescript-requiring-type-checking.js

```diff
@@ -77,6 +77,8 @@ module.exports = {
 
         '@typescript-eslint/no-unnecessary-type-arguments': 'error',
 
+        '@typescript-eslint/no-unsafe-argument': 'error',
```

Comment:
new rule from update [@typescript-eslint/no-unsafe-argument](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unsafe-argument.md)

Reply:
Huh, [we didn't know this wasn't in effect](https://github.com/ni-kismet/helium-security-ui/pull/421). It's very important when working with untyped imports.